### ISSUE TITLE
table.find usage in if statement

### ISSUE THREAD

**Issue by Synthranger** [reimakesgames/link#2]
https://github.com/reimakesgames/link/blob/106bc49cb26a500ce354089ca341a28b9213fe8b/src/shared/Link/Connection.lua?plain=1#L67-L67

`table.find` returns the index of the value which means it will never match with the Whitelist argument of the function since it is a boolean

Corrected function:
```lua
function Connection:FireSelectedClients(SelectedPlayers: Array<Player>, Whitelist: boolean, ...)
	IsServer("Connection:FireSelectedClients()")
	Whitelist = Whitelist or true
	for _, Player in Players:GetPlayers() do
		local Selected = typeof(table.find(SelectedPlayers, Player)) == "number"
		if Selected == Whitelist then
			self.__Remote:FireClient(Player, ...)
		end
	end
end
```

**Comment by reimakesgames** [comment]
thanks

**Comment by reimakesgames** [comment]
fixed with d4c0536
